_Boards/Atmel/Board_Mega/MFBoards.h

```diff
@@ -15,7 +15,7 @@
 
 #define MODULE_MAX_PINS 69
 #define MAX_OUTPUTS 40
-#define MAX_BUTTONS 67 // 69 max. numbering, 0/1 for serial
+#define MAX_BUTTONS 68
```

Comment:
This is 68 here: https://github.dev/MobiFlight/MobiFlight-Connector/blob/51613650fa8142f4a7338633dac631e2ec33fb2a/FirmwareSource/mobiflight/mobiflight.ino#L127